Vue/package.json

```diff
@@ -8,14 +8,15 @@
     "serve": "vite preview",
     "test": "playwright test",
     "prebuild": "playwright test",
-    "postinstall": "npx playwright install-deps",
+    "postinstall": "npx playwright install-deps && npm install @rollup/rollup-darwin-arm64",
```

Comment:
@Aeonitis The rollup dependency will break installs for other platforms. There are a few options that could work. 

- Create a new entry in the scripts object that will install this dependency 
- adding a step in the `_run_tests.sh` to check if the system is the Apple MChip and install the dependency
- you can globally install it on your system `npm i -g @rollup/rollup-darwin-arm64`